gridmap/job.py

```diff
@@ -848,11 +853,20 @@ def process_jobs(jobs, temp_dir='/scratch/', white_list=None, quiet=True,
     :param local: Should we execute the jobs locally in separate processes
                   instead of on the the cluster?
     :type local: bool
+    :param force_cluster: Should we raise an exception if access to cluster
+                          is not available?
+    :type force_cluster: bool
 
     :returns: List of Job results
     """
     if (not local and not DRMAA_PRESENT):
         logger = logging.getLogger(__name__)
+        if force_cluster:
+            logger.error(
```

Comment:
Since you're raising an exception, you don't really need to log this.  Instead just add the message to the exception:

```py
raise DRMAANotPresentException('Could not import drmaa, but cluster access required.)
```